afterpay/src/test/kotlin/com/afterpay/android/AfterpayCashAppApiTest.kt

```diff
@@ -0,0 +1,117 @@
+package com.afterpay.android
+import com.afterpay.android.cashapp.AfterpayCashAppApi
+import com.afterpay.android.cashapp.AfterpayCashAppApi.CashHttpVerb
+import com.afterpay.android.cashapp.AfterpayCashAppSigningResponse
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.MissingFieldException
+import org.junit.Assert.assertEquals
+import org.junit.BeforeClass
+import org.junit.Test
+import java.io.ByteArrayInputStream
+import java.io.ByteArrayOutputStream
+import java.io.InvalidObjectException
+import java.net.URL
+import javax.net.ssl.HttpsURLConnection
+
+class AfterpayCashAppApiTest {
+    companion object {
+        private val connection = mockk<HttpsURLConnection>()
+
+        @BeforeClass
+        @JvmStatic
+        fun setup() {
+            every { connection.requestMethod = "POST" } returns Unit
+            every { connection.doInput = true } returns Unit
+            every { connection.doOutput = true } returns Unit
+            every { connection.setRequestProperty(any(), any()) } returns Unit
+            every { connection.outputStream } returns ByteArrayOutputStream()
```

Comment:
Nit: I wonder if we used `relaxed = true` when defining the mock above, if we would still need to define all of this "stubs". Might be worth trying if that works, if so could remove this entire block on the `setup`.

eg.: `private val connection = mockk<HttpsURLConnection>(relaxed = true)`